modules/evm/src/lib.rs

```diff
@@ -344,6 +344,39 @@ decl_module! {
 			})
 		}
 
+		#[weight = T::GasToWeight::convert(*gas_limit)]
+		pub fn scheduled_call(
+			origin,
+			from: EvmAddress,
+			target: EvmAddress,
+			input: Vec<u8>,
+			value: BalanceOf<T>,
+			gas_limit: u32,
+			storage_limit: u32,
+		) -> DispatchResultWithPostInfo {
+			ensure_root(origin)?;
+
+			let from_account = T::AddressMapping::get_account_id(&from);
+			// unreserve the deposit for gas_limit and storage_limit
+			let total_fee = T::StorageDepositPerByte::get().saturating_mul(storage_limit.into()).saturating_add(gas_limit.into());
+			T::Currency::unreserve(&from_account, total_fee);
```

Comment:
Consider unreserve after the Runner `call` because it is fallible. Would not want to unreserve if the Runner::call fails and returns `Error::<T>::NoPermission` for example. Then, someone could call this without permission on purpose to unreserve that total fee amount.

Reply:
@xlc 
Maybe should remove unreserve and reserve. 
Just judge whether the free_balance >= total_fee

Reply:
The fee is not handle correctly here, but can fix it in later PR https://github.com/AcalaNetwork/Acala/issues/700